src/log/forwarding/log_forwarding_rules.py

```diff
@@ -188,9 +188,8 @@ def load_forwarding_rules_from_local_folder():
                         rule_obj = _create_log_forwarding_rule_object(rule)
                         log_forwarding_rules[bucket_name][rule['name']] = rule_obj
                     except IncorrectLogForwardingRuleFormat as ex:
-                        logger.warning(
-                            'Skipping incorrect log forwarding rule: %s in %s', rule, rule_file.name)
-                        logger.error(ex)
+                        logger.exception(
+                            f'Skipping incorrect log forwarding rule {rule} in {rule_file.name} {ex.message}')
```

Comment:
To consider if it would be valuable to see the whole stack when the debug logs are turned on. But I'm not 100% sure, as this "IncorrectLogForwardingRuleFormat" is only raised in a particular place in the code with the specific messages. So maybe the origin of the exception is always obvious and the stacktrace is unnecessary.

Reply:
i decide to not log exception in this case, thanks